src/item.rs

```diff
@@ -1179,7 +1190,13 @@ pub mod parsing {
         Ok(ArgCaptured {
             pat: input.parse()?,
             colon_token: input.parse()?,
-            ty: input.parse()?,
+            ty: if input.peek(Token![...]) {
+                Type::Verbatim(TypeVerbatim {
+                  tts: input.parse()?
```

Comment:
I don't think this is correct -- tts is a TokenStream so you would be slurping up the entire rest of the input as that TokenStream.

Separately, please check that your indentation is 4 spaces per Rust style.

```rust
fn f(a: ..., b: u8) {}
```

```rust
File {
    shebang: None,
    attrs: [],
    items: [
        Fn(
            ItemFn {
                attrs: [],
                vis: Inherited,
                constness: None,
                unsafety: None,
                asyncness: None,
                abi: None,
                ident: Ident {
                    ident: "f",
                    span: #0 bytes(97..98)
                },
                decl: FnDecl {
                    fn_token: Fn,
                    generics: Generics {
                        lt_token: None,
                        params: [],
                        gt_token: None,
                        where_clause: None
                    },
                    paren_token: Paren,
                    inputs: [
                        Captured(
                            ArgCaptured {
                                pat: Ident(
                                    PatIdent {
                                        by_ref: None,
                                        mutability: None,
                                        ident: Ident {
                                            ident: "a",
                                            span: #0 bytes(99..100)
                                        },
                                        subpat: None
                                    }
                                ),
                                colon_token: Colon,
                                ty: Verbatim(
                                    TypeVerbatim {
                                        tts: TokenStream [
                                            Punct {
                                                ch: '.',
                                                spacing: Joint,
                                                span: #0 bytes(102..105)
                                            },
                                            Punct {
                                                ch: '.',
                                                spacing: Joint,
                                                span: #0 bytes(102..105)
                                            },
                                            Punct {
                                                ch: '.',
                                                spacing: Joint,
                                                span: #0 bytes(102..105)
                                            },
                                            Punct {
                                                ch: ',',
                                                spacing: Alone,
                                                span: #0 bytes(105..106)
                                            },
                                            Ident {
                                                ident: "b",
                                                span: #0 bytes(107..108)
                                            },
                                            Punct {
                                                ch: ':',
                                                spacing: Alone,
                                                span: #0 bytes(108..109)
                                            },
                                            Ident {
                                                ident: "u8",
                                                span: #0 bytes(110..112)
                                            }
                                        ]
                                    }
                                )
                            }
                        )
                    ],
                    variadic: None,
                    output: Default
                },
                block: Block {
                    brace_token: Brace,
                    stmts: []
                }
            }
        )
    ]
}
```

Reply:
Good catch! I'll update this and if I can figure out the tests I might add a test.